extensions/spotify/init.lua

```diff
@@ -7,6 +7,21 @@ local spotify = {}
 local alert = require "hs.alert"
 local as = require "hs.applescript"
 
+--- hs.spotify.state_paused
+--- Constant
+--- A string that, when returned by hs.spotify.getPlayerState() indicates Spotify is paused
```

Comment:
Constants don't really need to specify their type, so this could just be "Returned by `hs.spotify.getPlayerState()` to indicate Spotify is paused" (note the use of backticks around the hs.spotify.getPlayerState() - we should really be doing that every time we refer to part of the API)